docs/usage/tutorials.md

```diff
@@ -617,3 +617,40 @@ You can then add the `<YOUR_DB_NAME>/` path to your nf-core/taxprofiler database
 </details>
 
 More information on custom KMCP database construction can be found [here](https://bioinf.shenwei.me/kmcp/database/#building-custom-databases).
+
+#### sylph custom database
+
+To build a sylph database, it requires only fasta files.
+
+```bash
+sylph sketch genomes/*.fa.gz
+```
+
+If all genomes are in the same file, you can run:
+
+```bash
+sylph sketch all_genomes.fa
+```
+
+You can then add the `<YOUR_DB_NAME>/` path to your nf-core/taxprofiler database input sheet.
+
+<details markdown="1">
+<summary>Expected files in database directory</summary>
+
+- `sylph`
+  - `.syldb`
+  </details>
+
+More information on custom sylph database construction can be found [here](https://sylph-docs.github.io/sylph-cookbook/#database-sketching-options).
+
+Classification using the sylph classifier consists of two steps: first running the `sylph profile` command, followed by the `sylph-tax taxprof` command.
+
+`sylph-tax taxprof` command requires a taxonomy file. You can create your own taxonomy metadata file. A taxonomic metadata file is a two-column TSV file:
+
+- Column 1: the name of your genome's FASTA file: my_mag.fa
+
+- Column 2: a semicolon-delimited taxonomy string. d**Archaea;p**Methanobacteriota_B;c**Thermococci;o**Thermococcales;f**Thermococcaceae;g**Thermococcus_A;s\_\_Thermococcus_A alcaliphilus
```

Comment:
Please give a realisitic example in acode block of this (and ideally with a couple of entries)

Reply:
Updated this part too.